src/digitalIdentity-frontend/src/app/pages/login-page/login-page.component.spec.ts

```diff
@@ -57,4 +72,42 @@ describe('LoginPageComponent', () => {
     let test_div = de.query(By.css('.test-card'));
     expect(test_div).toBeNull();
   });
+
+  it('should have an initial state like', () => {
```

Comment:
Description does not describe a test/behavior

Reply:
Does the test even make sense? If so, what would be an appropriate description?

Reply:
How can I tell whether your intended test is usefull? 
Nethertheless, this mini tests does not appear usefull